arm-automation/2015-10-31/swagger/account.json

```diff
@@ -0,0 +1,837 @@
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/UsageListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Sku": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Gets or sets the SKU name of the account.",
+          "enum": [
+            "Free",
+            "Basic"
+          ],
+          "x-ms-enum": {
+            "name": "SkuNameEnum",
+            "modelAsString": true
+          }
+        },
+        "family": {
+          "type": "string",
+          "description": "Gets or sets the SKU family."
+        },
+        "capacity": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Gets or sets the SKU capacity."
+        }
+      },
+      "description": "The account SKU."
+    },
+	"AutomationAccountUpdateProperties": {
+      "properties": {
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "Gets or sets account SKU."
+        }
+      },
+      "description": "The parameters supplied to the update account properties."
+    },
+    "AutomationAccountUpdateParameters": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/AutomationAccountUpdateProperties",
+          "x-ms-client-flatten": true,
+          "description": "Gets or sets account update properties."
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets or sets the name of the resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "Gets or sets the location of the resource."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets the tags attached to the resource."
+        }
+      },
+      "required": [
+        "properties"
+      ],
+      "description": "The parameters supplied to the update automation account operation."
+    },
+    "AutomationAccountProperties": {
+      "properties": {
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "Gets or sets the SKU of account."
+        },
+        "lastModifiedBy": {
+          "type": "string",
+          "description": "Gets or sets the last modified by."
+        },
+        "state": {
+          "type": "string",
+          "description": "Gets or sets the status of account.",
```

Comment:
Hi Vishrut,
 We have received exceptions for the below issues:
 [Azure Automation] validations (size, range, regexes) on properties and parameters missing  #1020
#1020
 [Azure Automation] Descriptions need to be corrected in all Azure Automation resources  #1021
#1021

Samer is aware of above issues. The two issues above will be revisited. These descriptions were inherited from Hyak specs and we won't be changing descriptions or validation regex at the moment
Leave a comment